src/libs/ReportUtils.ts

```diff
@@ -1234,14 +1234,14 @@ function isConciergeChatReport(report: OnyxInputOrEntry<Report>): boolean {
     return participantAccountIDs.has(CONCIERGE_ACCOUNT_ID_STRING);
 }
 
-function findSelfDMReportID(): string | undefined {
+function findSelfDMReport(): OnyxEntry<Report> {
     const allReports = ReportConnection.getAllReports();
     if (!allReports) {
         return;
     }
 
     const selfDMReport = Object.values(allReports).find((report) => isSelfDM(report) && !isThread(report));
-    return selfDMReport?.reportID;
+    return selfDMReport;
 }
```

Comment:
Why do we need this change?

Reply:
Let's revert this and use getReport(reportID) for the one case where you need the whole report